Tests/UtilityTests/ArgumentParserTests.swift

```diff
@@ -461,6 +474,30 @@ class ArgumentParserTests: XCTestCase {
             "",
             ""].joined(separator: "\n")))
     }
+    
+    func testRemainder() throws {
+        let parser = ArgumentParser(commandName: "SomeBinary", usage: "sample parser", overview: "Sample overview")
+        
+        let option = parser.add(option: "--foo", kind: String.self, usage: "The foo")
+        let positional = parser.add(positional: "executable", kind: [String].self, strategy: .remainder, usage: "A command to execute")
+        
+        let args = try parser.parse([
+            "--foo", "bar",
+            "exe", "--with", "options", "--foo", "notbar"
+        ])
+        
+        XCTAssertEqual(args.get(option), "bar")
+        XCTAssertEqual(args.get(positional) ?? [], ["exe", "--with", "options", "--foo", "notbar"])
+    }
+    
+    func testOverride() throws {
+        let parser = ArgumentParser(commandName: "SomeBinary", usage: "sample parser", overview: "Sample overview")
+        let option1 = parser.add(option: "--a", kind: Int.self)
+        let option2 = parser.add(option: "--a", kind: String.self)
```

Comment:
`SwiftPackageTool` depends on it because it re-adds `--version` to setup its mode: https://github.com/apple/swift-package-manager/blob/master/Sources/Commands/SwiftPackageTool.swift#L202

But we could fixed `SwiftPackageTool` to not do that.